dvc/lock.py

```diff
@@ -15,6 +16,12 @@
 
 DEFAULT_TIMEOUT = 5
 
+ERROR_MESSAGE = (
```

Comment:
I would consider renaming it, as ERROR_MESSAGE might be a bit unmeaningful, even considering its placement.

Reply:
@pared What do you suggest?

Reply:
address this comment? I think @pared meant something like "FAILED_TO_LOCK_MESSAGE"

Reply:
Consolidated in https://github.com/iterative/dvc/pull/2765#pullrequestreview-316606104.